python/ray/serve/controller.py

```diff
@@ -76,6 +83,386 @@ class Config:
         arbitrary_types_allowed = True
 
 
+@dataclass
+class ConfigurationStore:
+    backends: Dict[BackendTag, BackendInfo]
+    traffic_policies: Dict[EndpointTag, TrafficPolicy]
+    routes: Dict[BackendTag, Tuple[EndpointTag, Any]]
+
+    def __init__(self):
+        self.backends = dict()
+        self.traffic_policies = dict()
+        self.routes = dict()
+
+    def get_backend_configs(self) -> Dict[BackendTag, BackendConfig]:
+        result = {}
+        for tag, info in self.backends.items():
+            config = info.backend_config
+            result[tag] = config
+        return result
+
+    def get_backend(self, backend_tag: BackendTag) -> Optional[BackendInfo]:
+        return self.backends.get(backend_tag, None)
```

Comment:
```suggestion
        return self.backends.get(backend_tag)
```

`some_dict.get(key)` defaults to None